packages/web-components/src/components/rux-breadcrumb/test/breadcrumb.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { test, expect } from '../../../../tests/utils/_astro-fixtures'
+test.describe('Breadcrumb', () => {
+    test('it renders with breadcrumb items', async ({ page }) => {
+        const template = `
+        <rux-breadcrumb>
+            <rux-breadcrumb-item href="#" data-id="first">First</rux-breadcrumb-item>
+            <rux-breadcrumb-item href="#">Second</rux-breadcrumb-item>
+            <rux-breadcrumb-item href="#">Third</rux-breadcrumb-item>
+        </rux-breadcrumb>
+        `
+        await page.setContent(template)
+        const el = await page.locator('rux-breadcrumb')
+        const breadcrumbItem = el.locator(
+            `rux-breadcrumb-item[data-id='first']`
+        )
+        await expect(breadcrumbItem).toHaveClass('hydrated')
```

Comment:
we can nix this. all of our tests run through a fixture that will await and fail if the components aren't hydrated.